src/shared-styles/_typography.scss

```diff
@@ -35,6 +38,23 @@
   @include _typography($scale-point, $baseline-units, font.$secondary, 600);
 }
 
+@mixin _label-typography($color, $uppercase: true) {
+  color: $color;
+  @include _typography($scale-point: -3, $font-family: font.$secondary);
+  letter-spacing: font.$letterspacing-label;
+  @if $uppercase {
+    text-transform: uppercase;
+  }
+}
+
+@mixin label-content-typography($color: color.$text-secondary, $uppercase: true) {
```

Comment:
Shouldn't have `typography` in the name to avoid clumsy usage. Try renaming to `label-as-content`, so a usage would be `@include typography.label-as-content()`.